packages/contracts/contracts/optimistic-ethereum/OVM/bridge/tokens/OVM_L1ETHGateway.sol

```diff
@@ -68,66 +67,100 @@ contract OVM_L1ETHGateway is iOVM_L1ETHGateway, OVM_CrossDomainEnabled, Lib_Addr
      * Depositing *
      **************/
 
+    /**
+     * @dev This function can be called with no data
+     * to deposit an amount of ETH to the caller's balance on L2.
+     */
     receive()
         external
         payable
     {
-        _initiateDeposit(msg.sender, msg.sender);
+        _initiateDeposit(msg.sender, msg.sender, 0, bytes(""));
```

Comment:
Hmmm.  This is a nuance of removing the default gas value which I hadn't thought of.  I think this `recieve` will fail on the L2 side unless we specify a nonzero value as the gas parameter here.  I would be down to just do that; the `receive` is nice.  Can we add a quick integration test for using `recieve` so that we have coverage going forward too?

Reply:
FWIW, I did call that out above: https://github.com/ethereum-optimism/optimism/pull/824#discussion_r638849904. 
I guess it's easy to skim over comments on such a long PR review though.

Reply:
Yeah I see you did call it out, sorry for missing until this review!